[2025-05-01T15:18:30.927+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-01T15:18:30.942+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PlanningEngine.applySchemaMap manual__2025-05-01T15:11:04.438406+00:00 [queued]>
[2025-05-01T15:18:30.949+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PlanningEngine.applySchemaMap manual__2025-05-01T15:11:04.438406+00:00 [queued]>
[2025-05-01T15:18:30.951+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-01T15:18:30.964+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): applySchemaMap> on 2025-05-01 15:11:04.438406+00:00
[2025-05-01T15:18:30.970+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=198) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-01T15:18:30.972+0000] {standard_task_runner.py:72} INFO - Started process 199 to run task
[2025-05-01T15:18:30.972+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'PlanningEngine', 'applySchemaMap', 'manual__2025-05-01T15:11:04.438406+00:00', '--job-id', '3367', '--raw', '--subdir', 'DAGS_FOLDER/***DAG.py', '--cfg-path', '/tmp/tmp8_7ptdu8']
[2025-05-01T15:18:30.973+0000] {standard_task_runner.py:105} INFO - Job 3367: Subtask applySchemaMap
[2025-05-01T15:18:31.015+0000] {task_command.py:467} INFO - Running <TaskInstance: PlanningEngine.applySchemaMap manual__2025-05-01T15:11:04.438406+00:00 [running]> on host 29f5f6aefab8
[2025-05-01T15:18:31.080+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='PlanningEngine' AIRFLOW_CTX_TASK_ID='applySchemaMap' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T15:11:04.438406+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-01T15:11:04.438406+00:00'
[2025-05-01T15:18:31.081+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-01T15:18:31.122+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-01T15:18:31.129+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-01T15:18:31.130+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=PlanningEngine, task_id=applySchemaMap, run_id=manual__2025-05-01T15:11:04.438406+00:00, execution_date=20250501T151104, start_date=20250501T151830, end_date=20250501T151831
[2025-05-01T15:18:31.161+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-01T15:18:31.182+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-01T15:18:31.185+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
